code/js/localization/en.json

```diff
@@ -3,12 +3,34 @@
     "author": "Roman",
     "locale": "en"
   },
-  "calculator.title": "Calculator",
-  "calculator.h1": "Calculator by Roman",
+  "nav.button.home": "Home",
+  "nav.button.calculator": "Calculator",
+  "nav.button.practice": "Practice",
+  "nav.button.contact": "Contact",
+
+  "nav-language-switcher-button": "Language",
+  "nav-language-switcher-english": "English",
+  "nav-language-switcher-czech": "Czech",
+
+  "index.title": "Home",
+  "index.h2": "Hi, I'm Roman Hvizdak",
+  "index.h3": "I present my site where the calculator is located, a page with generating examples for practice and the possibility to print a set of examples",
+  "index.square.p": "More about this project on GitHub",
```

Comment:
What is it: index.square.p ??? Be more concrete also for all headlines h1,h2 ...

Reply:
index.square.p renamed to index.square.github.paragraph... and all done for headlines also